spec/features/app_spec.rb

```diff
@@ -513,3 +513,34 @@
     end
   end
 end
+
+# we test the metadata APIs
+describe 'metadata APIs' do
+  describe 'providers list' do
+    it 'responds successfully' do
+      get "v1/providers"
+      expect(last_response.status).to eq 200
+    end
+  end
+
+  describe 'server types list' do
+    it 'responds successfully' do
+      get "v1/#{$valid_providers.first}/servers/types"
+      expect(last_response.status).to eq 200
```

Comment:
Again, the return value is important, not just that the server responds with 200.